### ISSUE TITLE
Updating manga list can't be saved

### ISSUE THREAD

**Issue by Slasar41** [dazedcat19/FMD2#1]
I use the converter tool and all seems fine, I can double click the title on manga list and manga info will load fine and the list will still exist after reopening FMD.
But when I update the manga list, FMD can't load the manga info anymore and reopening FMD will remove the list.

**Comment by dazedcat19** [comment]
Sorry, I forgot to make a proper change to mangalist db.

Can you convert all the dbs tho? Upload it to a new repo.

**Comment by Slasar41** [comment]
Can I just download all the available, convert it, and then upload to this?
Ah yeah, I still don't know the place to upload the databases, where is it?
Also, how to get the generated website ID for new website?

**Comment by dazedcat19** [comment]
https://github.com/dazedcat19/FMD2-DB/
Put them in 7z folder. Following config.json

Yes, just use a batch script to rename them.
https://github.com/dazedcat19/FMD2/tree/master/scripts

There's some websites not converted yet. Like kissmanga. I'm reluctant to rewrite it. The quality isn't that good. but too much security. Madokami etc that broke also.

**Comment by Slasar41** [comment]
Alright. Should we have the statistics too like in the old repo?

There's many people using KissManga, you can see the latest kissmanga issue on the old repo.
Also, 90% of my favorites is from KissManga 😅 so I can't use this for my main FMD yet.

**Comment by dazedcat19** [comment]
kissmanga module was very simple. except the encryption part. I think there's pure lua encryption library out there that can be used. I'm reluctant to add more encrytion apis. there's way too many variants. too much work. better use what available.

**Comment by terabyte-gc** [comment]
I just updated/converted my files to FMD2 today using **converter_x86_64-win64.7z** from **release 2.0.21.0** and I'm facing the same issue. Trying to add/remove any manga from the favorite list doesn't save the change once the application is closed. I do still have the old database files in case there is a need to re-convert them.

(As a test I opened the **favorites.db** file with a SQLite editor and manually removing a specific record worked just fine.)
